package-lock.json

```diff
@@ -14464,6 +14426,7 @@
     },
     "transliteration": {
       "version": "github:reactioncommerce/transliteration#699d48cc8dd9a64f1a2773e1b36b6faa4bbdca2f",
+      "from": "transliteration@github:reactioncommerce/transliteration#699d48cc8dd9a64f1a2773e1b36b6faa4bbdca2f",
```

Comment:
any idea why this is changing?

Reply:
No, but it seems to be fixing an issue we are having with `transliteration` when doing builds. But it just popped up when I regened this file after reinstall modules. We could revert it since it's not directly related to this PR

Reply:
seems like we should have this somewhere then, but I agree it seems tangential to this PR.

Reply:
I noticed this before and I think it's based on which NPM version is used when you install. Maybe you used npm6?